src/main/py/hl7_listener/main.py

```diff
@@ -106,27 +150,39 @@ async def hl7_receiver():
         ) as hl7_server:
             # Listen forever or until a cancel occurs.
             await hl7_server.serve_forever()
+
     except asyncio.CancelledError:
         # Cancel errors are expected.
-        logger.info(logging_codes.HL7_MLLP_RECEIVER_CANCELLED)
+        logger.info(
+            HL7_MLLP_RECEIVER_CANCELLED,
+            logging_code="HL7LLOG005"
+        )
         pass
+
     except Exception as exp:
-        logger.error(logging_codes.HL7_MLLP_RECEIVER_ERR, exc_info=Exception(exception_formatter(str(exp))))
+        logger.error(
+            HL7_MLLP_RECEIVER_ERR,
+            logging_code="HL7LERR005",
+            exception=Exception(exception_formatter(str(exp)))
+        )
         raise Exception(exception_formatter(str(exp)))
 
 
 async def main():
+    # Create the logger and add the correlation_id to the logs.
+    logs_inject_correlation_id(logger)
     logger.info(
-        logging_codes.STARTUP_ENV_VARS,
-        json.dumps({
-            "settings": settings.dict(),
-            "messager_settings": messager_settings.dict()
-        })
+        STARTUP_ENV_VARS,
```

Comment:
One thing to be aware of is that some settings may leak connection information. Pydantic support [secret fields](https://docs.pydantic.dev/1.10/usage/types/#secret-types) which are quite helpful.

Here's an [example](https://github.com/coverahealth/qcc-study-hall/blob/main/study-index-task/src/qcc_study_hall/study_index/settings.py#L31-L33) from `qcc-study-hall`.